gradle/libs.versions.toml

```diff
@@ -1,4 +1,5 @@
 [versions]
+ebay-graphAnalytics = "0.0.1"
```

Comment:
I noticed that you have dependabot setup for this project,  Is this going to be added automatically?

Reply:
It should, yes.  By default dependabot includes all libs.